### ISSUE TITLE
Keepalive connections not enabled in nginx.conf

### ISSUE THREAD

**Issue by makhomed** [getsentry/self-hosted#2143]
### Self-Hosted Version

23.4.0

### CPU Architecture

x86_64

### Docker Version

23.0.3

### Docker Compose Version

2.12.2

### Steps to Reproduce

Install self-hosted sentry 23.4.0

### Expected Result

Keepalive connections between nginx and backends are enabled.

### Actual Result

Keepalive connections between nginx and backends are disabled.

fragment of self-hosted-23.4.0/nginx/nginx.conf:
```nginx
        proxy_http_version 1.1;

        # Remove the Connection header if the client sends it,
        # it could be "close" to close a keepalive connection
        proxy_set_header Connection '';

        upstream relay {
                server relay:3000;
        }

        upstream sentry {
                server web:9000;
        }
```
To enable keepalive connections you should add ```keepalive 32;``` or ```keepalive 64;``` to each upstream block.

Detailed description from nginx.com website:

https://www.nginx.com/blog/avoiding-top-10-nginx-configuration-mistakes/#no-keepalives

**Comment by aldy505** [comment]
It'll be on the next release. See https://github.com/getsentry/self-hosted/pull/2099

**Comment by hubertdeng123** [comment]
Yep, now should be in [23.5.0](https://github.com/getsentry/self-hosted/releases/tag/23.5.0)

**Comment by makhomed** [comment]
> Yep, now should be in [23.5.0](https://github.com/getsentry/self-hosted/releases/tag/23.5.0)

Yes, this bug is fixed in the 23.5.0 release.
